src/main/java/br/com/zup/estrelas/financas/entity/Sugestao.java

```diff
@@ -0,0 +1,60 @@
+package br.com.zup.estrelas.financas.entity;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+
+
+@Entity
+public class Sugestao {
+
+
+    @Column(name = "id_sugestao")
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long idSugestao;
+
+    @Column
+    private String titulo;
+
+    @Column
+    private String descricao;
+
+    @Column
+    private TipoSugestao tipoSugestao;

```

Comment:
This needs te property name @ Column like u did on line 14 to be snake_case on database